homeassistant/components/google_assistant/helpers.py

```diff
@@ -612,10 +611,6 @@ def sync_serialize(self, agent_user_id, instance_uuid):
                 "webhookId": self.config.get_local_webhook_id(agent_user_id),
                 "httpPort": self.hass.http.server_port,
```

Comment:
Why do we still need httpPort? I assume we do discovery of the instance using mdns? so it includes the port too no?

Reply:
I guess parsing the port out of the internal URL might be better

Reply:
Yup agree.